ck.sh

```diff
@@ -59,7 +59,7 @@ case "$COMMAND" in
 				cd ckeditor-dev
 
 				# Convert svg icons to png
```

Comment:
I assume this comment is no longer acurate?

Reply:
Upsy